src/FriendsPage.tsx

```diff
@@ -81,50 +83,42 @@ const FriendsPage: React.FC = () => {
           <button>Invite!</button>
         </AutoTable>
       }
+      error={error}
+      loaded={isLoaded}
     >
-      {!isLoaded ? (
-        "Loading..."
-      ) : error ? (
-        "An error occured."
-      ) : (
```

Comment:
The only real difference is that I removed the loading and error logic from this page